.gitignore

```diff
@@ -63,3 +63,4 @@ pipeline/vault_cre*
 pipeline/custom-*
 
 dist/
+pkg/connectors/datacatalog/openapiclient/.openapi-generator/FILES
```

Comment:
do we need this entry ?

Reply:
It may be needed in the scenario where we update the openapi spec and generate the client code again. In this case the file FILES will be recreated and we may mistakenly checkin the file. So it may be better to add this file in the gitignore.